addon/lib/metrics.js

```diff
@@ -121,13 +151,32 @@ module.exports = {
     this.updateExperiment(addonId, {features: features});
   },
 
+  onReceiveVariantDefs: function(ev) {
+    if (!store.experimentVariants) {
+      store.experimentVariants = {};
+    }
+
+    const { subject, data } = ev;
+    const dataParsed = variantMaker.parseTests(JSON.parse(data));
+
+    store.experimentVariants[subject] = dataParsed;
+    Events.emit(EVENT_SEND_VARIANTS, {
+      data: JSON.stringify(dataParsed),
+      subject: self.id
+    });
+  },
+
   onExperimentPing: function(ev) {
     const { subject, data } = ev;
     const dataParsed = JSON.parse(data);
 
     // TODO: Map add-on ID (subject) to other pingTypes as necessary
     const pingType = 'testpilottest';
 
+    if (subject in store.experimentVariants) {
```

Comment:
We didn't specify this data in the `testpilottest` ping - though that's a good place for it. 

But, we *did* (loosely) specify it [in the main `testpilot` ping under the `features` key for each listed experiment](https://github.com/mozilla/testpilot/blob/master/docs/README-METRICS.md#data-collection-details). We should add this data to the main ping - either when [an experiment is updated in the ping](https://github.com/mozilla/testpilot/blob/master/addon/lib/metrics.js#L105) or at the [time of submission](https://github.com/mozilla/testpilot/blob/master/addon/lib/metrics.js#L97).